test-files/cmd/slow/PowerLineDiffConflateTest.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```

Comment:
added diff conflate tests for each of the linear data types that didn't already have one